src/parquet/encodings/plain-encoding.h

```diff
@@ -52,6 +52,19 @@ class PlainDecoder : public Decoder {
     return GetValues(buffer, max_values, sizeof(int64_t));
   }
 
+  virtual int GetInt96(Int96* buffer, int max_values) {
+    max_values = std::min(max_values, num_values_);
+    int size = 3*sizeof(uint32_t);//Struct Int96 could be padded
```

Comment:
If we decide to portably pack `Int96` we can use this code from flatbuffers

https://github.com/google/flatbuffers/blob/master/include/flatbuffers/flatbuffers.h#L1355